code/datums/status_effects/debuffs/strandling.dm

```diff
@@ -0,0 +1,106 @@
+	return TRUE
+
+/datum/status_effect/strandling/on_remove()
+	UnregisterSignal(owner, list(COMSIG_CARBON_PRE_BREATHE, COMSIG_ATOM_TOOL_ACT(TOOL_WIRECUTTER), COMSIG_CARBON_PRE_HELP_ACT))
+
+/datum/status_effect/strandling/get_examine_text()
+	return span_warning("[owner.p_they(TRUE)] seem[owner.p_s()] to be being choked by some durathread strands. You may be able to <b>cut</b> them off.")
+
+/// Signal proc for [COMSIG_CARBON_PRE_BREATHE], causes losebreath whenever we're trying to breathe
+/datum/status_effect/strandling/proc/on_breathe(mob/living/source)
+	SIGNAL_HANDLER
+
+	if(source.getorganslot(ORGAN_SLOT_BREATHING_TUBE))
+		return
+
+	source.losebreath++
+
+/// Signal proc for [COMSIG_ATOM_TOOL_ACT] with [TOOL_WIRECUTTER], allowing wirecutters to remove the effect (from others / themself)
+/datum/status_effect/strandling/proc/on_cut(mob/living/source, mob/user, obj/item/tool)
+	SIGNAL_HANDLER
+
+	if(DOING_INTERACTION(user, REF(src)))
+		return
+
+	INVOKE_ASYNC(src, .proc/try_remove_effect, user, tool)
+	return COMPONENT_BLOCK_TOOL_ATTACK
+
+/// Signal proc for [COMSIG_CARBON_PRE_HELP_ACT], allowing someone to remove the effect by hand
+/datum/status_effect/strandling/proc/on_self_check(mob/living/carbon/source, mob/living/helper)
+	SIGNAL_HANDLER
+
+	if(DOING_INTERACTION(helper, REF(src)))
+		return
+
+	INVOKE_ASYNC(src, .proc/try_remove_effect, helper)
+	return COMPONENT_BLOCK_HELP_ACT
+
+/**
+ * Attempts a do_after to remove the effect and stop the strangling.
+ *
+ * user - the mob attempting to remove the strangle. Can be the same as the owner.
+ * tool - the tool the user's using to remove the strange. Can be null.
+ */
+/datum/status_effect/strandling/proc/try_remove_effect(mob/user, obj/item/tool)
+	if(user.incapacitated() || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
+		return
+
+	user.visible_message(
+		span_notice("[user] attempts to [tool ? "cut":"remove"] the strand from around [owner == user ? "[owner.p_their()]":"[owner]'s"] neck..."),
+		span_notice("You attempt to [tool ? "cut":"remove"] the strand from around [owner == user ? "your":"[owner]'s"] neck..."),
+	)
+
+	// Play a sound if we have a tool
+	tool?.play_tool_sound(owner)
+
+	// Now try to remove the effect with a doafter. If we have a tool, we'll even remove it 60% faster.
+	if(!do_mob(user, owner, time_to_remove * (tool ? STRANGLING_TOOL_MULTIPLIER : 1), interaction_key = REF(src)))
+		to_chat(user, span_warning("You fail to [tool ? "cut":"remove"] the strand from around [owner == user ? "your":"[owner]'s"] neck!"))
+		return FALSE
+
+	// Play another sound after we're done
+	tool?.play_tool_sound(owner)
+
+	user.visible_message(
+		span_notice("[user] successfully [tool ? "cut":"remove"] the strand from around [owner == user ? "[owner.p_their()]":"[owner]'s"] neck."),
+		span_notice("You successfully [tool ? "cut":"remove"] the strand from around [owner == user ? "your":"[owner]'s"] neck."),
+	)
+	qdel(src)
+	return TRUE
```

Comment:
I'm really kinda meh about checking tool's existence 8 times in this proc but honestly i don't have a better solution so this is fine